packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -261,7 +261,7 @@ $ex
       try {
         await window.setLocation(const math.Point<int>(0, 0));
         await window.setSize(math.Rectangle<int>(0, 0, x, y));
-      } catch (_) {
+      } on Exception catch (_) {
```

Comment:
nit we don't need to catch unused Exception

Reply:
Done